hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -297,6 +297,8 @@ def get_different_seeds_hyperdrive_config(self) -> HyperDriveConfig:
 
 
 class DatasetParams(param.Parameterized):
+    input_datasets: List[str] = param.List(default=[], class_=str, doc="Do not use. For AML SDK v2 only")
+    datastore_name: str = param.String(default="", doc="Datastore to look for data in")
```

Comment:
this might be used on the commandline, considering shortening to `datastore`?

Reply:
having said that, the more generic sthing would be to to extend azuredatasets to allow datasets to look like `datastore:datasetname` (after all, they might live on differnt datastores)

Reply:
I think searching through all datastores may be outside the scope of this PR. For now I have shortened the arg to datastore